preprocessing/DataCleanser.py

```diff
@@ -1,7 +1,12 @@
+import nltk
+from nltk.corpus import stopwords
+nltk.download('punkt')
```

Comment:
This only needs to be downloaded once, so I don't think we need it on our code. Maybe just comment it out and say if needed, uncomment to download. Same applies to stopwords.
Also, what does punkt do? I get downloading the stop words, but not this.